### DISCUSSION TITLE
Wie ist die HW Verbindung Viessmann Anlage / RPI?

### DISCUSSION THREAD

**Discussion by neandr** [MyHomeMyData/E3onCAN#3] [Ideas]
Wie wird der RPI mit der ViAnlage verbunden? Gibt es einen speziellen Adapter? 
Es ist reizvoll jenseits der Viessmann API direkten Zugriff (per MQTT?) auf das System zu bekommen.
Bitte um Referenz/Link der erforderlichen HW.

**Comment by MyHomeMyData** [comment]
Sorry, habe die Anfrage übersehen. Viele Infos zu open3e gibt es [hier](https://github.com/open3e/open3e). Da gibt es auch ein Wiki mit Anwendungsbeispielen.
